Gopkg.toml

```diff
@@ -23,19 +23,41 @@ required = [
   version = "v1.15.5"
 
 [[override]]
-  name = "sigs.k8s.io/cluster-api"
-  branch = "master"
+  name = "k8s.io/kube-aggregator"
+  branch = "release-1.10"
 
 [[override]]
-  #branch = "master"
-  name = "github.com/openshift/cluster-api-actuator-pkg"
-  revision = "6983d93876009e8c004dfe7868c1dece4a9d1137"
+  name = "github.com/openshift/cluster-api"
+  revision = "91fca585a85b163ddfd119fd09c128c9feadddca"
 
-# STANZAS BELOW ARE GENERATED AND MAY BE WRITTEN - DO NOT MODIFY BELOW THIS LINE.
+[[override]]
+  name = "k8s.io/code-generator"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "6702109cc68eb6fe6350b83e14407c8d7309fd1a"
 
-[[constraint]]
-  name="sigs.k8s.io/controller-runtime"
-  version="v0.1.1"
+[[override]]
+  name = "k8s.io/api"
+  # revision for tag "kubernetes-1.11.2"
```

Comment:
version = "kubernetes-1.11.2" does not work? The same for other `override`'s cases.

Reply:
this is consolidating with mao. In a follow up we need to pin all to 1.12.*